lib/models/installation.js

```diff
@@ -95,11 +95,12 @@ class Installation extends Sequelize.Model {
         jiraHost: payload.host,
       }).then(async (record) => {
         const subscriptions = await Subscription.getAllForClientKey(record.clientKey);
-        subscriptions.forEach(subscription => {
-          subscription.update({
+        for (const subscription of subscriptions) {
+          await subscription.update({
             jiraHost: record.jiraHost,
           });
-        });
+        }
```

Comment:
I actually was thinking, if there's a lot of subscriptions, it would be slower than having everything in parallel since the loop would do everything sequentially.  An alternative would be to use `Promise.all` :)

```
await Promise.all(
            subscriptions.map(
                subscription => subscription.update({jiraHost: record.jiraHost})
            )
        );
```

Reply:
Good point. Let's not make that slower.